bigframes/dataframe.py

```diff
@@ -3027,6 +3027,7 @@ def to_gbq(
         index: bool = True,
         ordering_id: Optional[str] = None,
         clustering_columns: Union[pandas.Index, Iterable[typing.Hashable]] = (),
+        labels: dict[str, str] = {},
```

Comment:
Please update the doc of this function too:
https://github.com/googleapis/python-bigquery-dataframes/blob/main/third_party/bigframes_vendored/pandas/core/frame.py#L385

Reply:
done!